src/common/offsets.h

```diff
@@ -155,3 +158,14 @@ const u64 CRC_AUDIT_TASK_INFO_LOGINUID = 0xe6dd107ea26da1a0;
 
 /* LINUX_KERNEL_VERSION */
 const u64 CRC_LINUX_KERNEL_VERSION = 0xbd0860b98f6d2ade;
+
+/* BTRFS support */
+
+/* btrfs_inode->root */
+const u64 CRC_BTRFS_INODE_ROOT = 0xb45c091ac97692c3;
+
+/* btrfs_inode->vfs_inode */
+const u64 CRC_BTRFS_INODE_VFS_INODE = 0xc4a24e4c91af5451;
+
+/* btrfs_root->anon_dev */
+const u64 CRC_BTRFS_ROOT_ANON_DEV = 0xa274b01a7473b236;
```

Comment:
nit: let's move this above the `LINUX_KERNEL_VERSION` crc because it's an offset like the rest. the `LINUX_KERNEL_VERSION` is more of a hack so it makes sense to be separate.